Pod/Classes/public/WistiaAPI.swift

```diff
@@ -105,11 +107,26 @@ extension WistiaAPI {
         Alamofire.request(.GET, "\(WistiaAPI.APIBaseURL)/account.json", parameters: params)
             .responseJSON { response in
 
-                if let JSON = response.result.value as? [String: AnyObject],
-                    account = ModelBuilder.accountFromHash(JSON) {
-                    completionHander(account: account)
-                } else {
-                    completionHander(account: nil)
+                // 'Swift-like' way to handle Alamofire responses
+                switch response.result {
+                    
+                    // Success! Let's parse that JSON.
```

Comment:
When the code is sufficiently self-documenting, we don't need additional comments.  Indeed, it's safer not to have them as they can get stale.

I would drop this one and some of the others added in this class.